pandas/core/arrays/integer.py

```diff
@@ -261,6 +265,11 @@ def __init__(self, values, mask, copy=False):
     def _from_sequence(cls, scalars, dtype=None, copy=False):
         return integer_array(scalars, dtype=dtype, copy=copy)
 
+    @classmethod
+    def _from_sequence_of_strings(cls, strings, dtype=None, copy=False):
+        scalars = to_numeric(strings, errors='raise')
```

Comment:
we need to ensure here that the result are integers I think? (or raise an error if that is not the case)

Reply:
I believe that between the `errors='raise'` and the subsequent call to `cls._from_sequence` would raise the appropriate error.